src/routes/blog/developer-journey/timeleap-protocol.tiramisu

```diff
@@ -0,0 +1,143 @@
+title { Timeleap: The Communication Protocol }
+
+So, we've talked about Timeleap giving you the tools to create a distributed
+computation network. In this article, we are connecting the dots: Timeleap, Sia,
+and the rules of the network. What does every network have? A protocol!
+
+Think of protocol as the complete rule book for every communication in the
+network. Timeleap has created its communication protocol to ensure that every
+message is reliable, secure, and fast. And as it was mentioned before, you don't
+have to build the protocol from scratch. Our broker-worker model has already
+implemented it for you.
```

Comment:
This paragraph is massively misleading and uses wrong terminology. Some points:

1. A protocol is like a blueprint for implementing something; we have a reference implementation of this blueprint (Timeleap broker/worker, the Go project). Others can study the blueprint and implement their own.

2. The protocol does more than make sure every message is reliable, secure, and fast; it also ensures reproducibility and compatibility in different implementations. Also, it specifies more things than just about messages.